### PR TITLE
Add function formatForeignCurrency

### PR DESCRIPTION
**Pull request by Ben305**
This PR adds a function `formatForeignCurrency`

This can be used to format an amount in the format specified by the current locale, but with a different currency symbol.

For example, a german user wants to see `1220,35 $` instead of `$1220.35`.

@BenjaminVanRyseghem What do you think?

### CODE REVIEW

File: numbro.js [hunk 1]
```diff
@@ -1149,7 +1153,16 @@
...
+                cultures[currentCulture].currency.symbol,
+                inputString ? inputString : defaultCurrencyFormat,
+                (roundingFunction !== undefined) ? roundingFunction : Math.round
+            );
+        },
+
+        formatForeignCurrency: function(inputString, currencySymbol, roundingFunction) {
+            return formatForeignCurrency(this,
                 inputString ? inputString : defaultCurrencyFormat,
+                currencySymbol,
```
Comment (BenjaminVanRyseghem) [inline numbro.js]: looks like the arguments are in wrong order, or?

shouldn't it be

```js
formatForeignCurrency(this, currencySymbol, inputString, ...)
```

Comment (Ben305) [inline numbro.js]: You're right, I've fixed this and have extended the tests for the new function.

Comment (BenjaminVanRyseghem) [inline numbro.js]: :+1:

Comment (Ben305) [inline numbro.js]: It would be great if this and my other PR would make it into a new release :-)

File: tests/numbro/format.js [hunk 2]
```diff
@@ -211,6 +211,79 @@ exports.format = {
...
+            },
+            defaults: {
+                currencyFormat: '(0.00a)'
+            }
+        });
+
+        numbro.culture('test1');
+
+        var tests = [
+                [1000.234,'€','$0,0.00','€1,000.23'],
```
Comment (BenjaminVanRyseghem) [inline tests/numbro/format.js]: your culture is postfix, so why isn't the output `'1,000.23€'`?

**Comment by BenjaminVanRyseghem** [comment]
in numbro 1.9.0

**Pull request merged**